tests/Go/PhpUnit/ClassMemberNotWovenConstraint.php

```diff
@@ -41,9 +37,13 @@ public function matches($other)
             throw new \InvalidArgumentException(sprintf('Expected instance of "%s", got "%s".', ClassAdvisorIdentifier::class, is_object($other) ? get_class($other) : gettype($other)));
         }
 
-        $wovenAdvisorIdentifiers = $this->getWovenAdvisorIdentifiers($other->getClass());
+        $reflectionClass         = ReflectionClass::create($other->getClass(), $this->configuration);
```

Comment:
Hmm, what is this? I can't remember `::create()` method in the `ReflectionClass`.

**UPD1** Ah, ok, you have introduced new helper class for that. Maybe rename it to more meaningful name to prevent collision with native `ReflectionClass` and same class from `goaop/parser-reflection`? I would vote for `ProxyClassReflectionHelper` or similar.